Plugins/Wox.Plugin.Program/packages.config

```diff
@@ -3,5 +3,4 @@
   <package id="JetBrains.Annotations" version="10.3.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
   <package id="System.Runtime" version="4.0.0" targetFramework="net452" />
-  <package id="UwpDesktop" version="10.0.14393.3" targetFramework="net452" />
```

Comment:
Please reverse the UWPDesktop Nuget package removable. This is required for Program plugin to work properly. May consider another option in the future as discussed in some PRs in the upstream repo.